features/step_definitions/et1_ccd_steps.rb

```diff
@@ -112,7 +115,10 @@
 Then /^the RTF file should be present in CCD$/ do
   admin_api = EtFullSystem::Test::AdminApi.new atos_interface: atos_interface
   reference_number = admin_api.get_reference_number(claim_application_reference: @claim_application_reference)
-  ccd_object = EtFullSystem::Test::Ccd::Et1CcdSingleClaimant.find_by_reference(reference_number)
+  office_code = @respondent[0]["expected_office"]
+  ccd_office_lookup = ::EtFullSystem::Test::CcdOfficeLookUp
+  ccd_object = EtFullSystem::Test::Ccd::Et1CcdSingleClaimant.find_by_reference(reference_number, ccd_office_lookup.office_lookup[:single][office_code])
+  
```

Comment:
Why not pass office code into the helper object ? Might make these a little easier to read and all the lookup stuff will be done in the object